### PR TITLE
EdDSA signer cleanup

### PR DESCRIPTION
**Pull request by fredmorcos**
### Short description
Cleanup (formatting, lints) of the EdDSA signer. Mostly automated changes.

### Checklist
I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [X] compiled this code
- [X] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: pdns/opensslsigners.cc [hunk 1]
```diff
@@ -1828,15 +1824,15 @@ class OpenSSLEDDSADNSCryptoKeyEngine : public DNSCryptoKeyEngine
   size_t d_len{0};
   int d_id{0};
 
-  std::unique_ptr<EVP_PKEY, void(*)(EVP_PKEY*)> d_edkey;
+  std::unique_ptr<EVP_PKEY, void (*)(EVP_PKEY*)> d_edkey;
```
Comment (rgacogne) [inline pdns/opensslsigners.cc:1827]:
```suggestion
  std::unique_ptr<EVP_PKEY, decltype(&EVP_PKEY_free)> d_edkey;
```

**Review by rgacogne — APPROVED** [review]
One non-blocking nit, looks good to me otherwise!

File: pdns/opensslsigners.cc (hunk 1 above)
Comment (fredmorcos) [reply pdns/opensslsigners.cc:1827]: Just `Key` should work here, actually. Thanks!

**Comment by fredmorcos** [comment]
Reminder to merge :-)

**Pull request merged**